Frontend/CO.CDP.OrganisationApp/appsettings.Development.json

```diff
@@ -6,6 +6,7 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "GoogleTagId": "G-2TLMW60D58",
```

Comment:
This will result in tracking our local development environments. Is that what we want?

Reply:
I'm also not fan of committing the ID in the codebase. 
hence the [original thought about introducing an explicit](https://goacoglobal.slack.com/archives/C06M34UGL59/p1723803380745789) env_var for it, instead of being rely on the `ASPNETCORE_ENVIRONMENT` and `appsettings`

Reply:
Variable set based on `ASPNETCORE_ENVIRONMENT`  is a recommend approach for asp.net application

Reply:
We got only one google tag for the rest of the environments, with my previous experience GTM tag's were assigned for each environment including development. @jakzal what is your approach regarding development?

Reply:
@webit4me apologies but I am unsure what you mean? 
Are you suggesting not to hardcode in codebase or store in AWS only? or am I missing the point completely here.